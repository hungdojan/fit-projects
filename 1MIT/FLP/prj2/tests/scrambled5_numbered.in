653
154
154
425 332 641 455
614 625 332 143
114 615 332 166
243
266
522
