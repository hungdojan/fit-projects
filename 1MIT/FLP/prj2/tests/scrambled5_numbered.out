653
154
154
425 332 641 455
614 625 332 143
114 615 332 166
243
266
522

253
254
154
625 332 645 114
114 625 332 645
114 615 332 635
443
666
122

344
555
221
114 625 332 645
114 625 332 645
114 615 332 635
443
666
122

555
555
221
114 622 333 445
114 622 333 445
114 611 222 335
443
666
666

555
555
555
111 222 333 444
111 222 333 444
444 111 222 333
666
666
666

555
555
555
111 222 333 444
111 222 333 444
111 222 333 444
666
666
666
